src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -1,2 +1,3 @@
+<button (click)="editLayoutGrid.addBox()">Add Box</button>
```

Comment:
I move this because:

1. We need it here
2. We need to fix the action-button and that weird issue with the command and the closure.